python/hail/methods/statgen.py

```diff
@@ -2973,7 +2973,7 @@ def ld_prune(call_expr, r2=0.2, bp_window_size=1000000, memory_per_core=256):
     Returns
     -------
     :class:`.Table`
-        Table of variants after pruning.
+        Table of variants in linkage disequilibrium.
```

Comment:
This is wrong. Linkage disequilibrium means the opposite of what you intended. I think this is clear: `Table with the maximally independent set of variants.` Can you also fix the first line of your changes above?